pycytominer/cyto_utils/cells.py

```diff
@@ -429,13 +429,17 @@ def get_sql_table_col_names(self, table):
 
         return meta_cols, feat_cols
 
-    def load_compartment(self, compartment):
+    def load_compartment(self, compartment, float_datatype: type = np.float64):
```

Comment:
I think most folks will use this after initializing the SingleCells class. So it's probably worth making this an init argument and using `self.single_cell_merge_float_datatype` throughout (note the rename - maybe there is something better to rename it to?)

Reply:
This makes a lot of sense, thank you for this feedback. Pondering on this makes me wonder if perhaps more than floating point data (for example, int's) may need or benefit from a default. I'll add in an attribute for `SingleCells` float datatypes with this in mind.

Reply:
> Pondering on this makes me wonder if perhaps more than floating point data (for example, int's) may need or benefit from a default.

Yeah, it likely would! IIRC we had discussed this previously (creating some sort of feature dictionary) but decided to leave this for a future research project. 

My sense is to focus on `float_datatype` here, since we've seen such huge performance boosts and then worry about this dictionary in a future PR